documentation/README.adoc

```diff
@@ -21,6 +23,45 @@ implementation("io.github.logrecorder:logrecorder-api:${logrecorder.version}")
     <groupId>io.github.logrecorder</groupId>
     <artifactId>logrecorder-api</artifactId>
     <version>${logrecorder.version}</version>
+    <scope>test</scope>
+</dependency>
+----
+
+=== Framework Support: Junit5
+
+.Gradle
+[source,groovy]
+----
+testImplementation("io.github.logrecorder:logrecorder-common-junit5:${logrecorder.version}")
```

Comment:
Maybe we should rename the `logrecorder-common-{{testframework}}` to `logrecorder-{{testframework}}` since in order to get LogRecorder to work, users need to declare two dependencies now:
1) The matching Test-Framework
2) The matching Logging-Framework

Reply:
I thought the same while typing the documentation. just thought i get feedback on that^^